app/views/webview/drug_stocks/new.html.erb

```diff
@@ -16,14 +16,14 @@
     <div class="progress-contents">
 
       <div id="drug-stock-form">
-      
+
         <style>
           label.sr-only {display:none;}
           .form-group {padding-left:0;padding-top:4px;padding-bottom:32px;}
         </style>
-      
+
         <div class="progress-contents">
-          <a href="#" onclick="closeWindow('drug-stock-form', 'progress-start'); return false" class="help-back">
+          <a href="simple://progress-tab" title="Go Back" class="help-back">
```

Comment:
Do we need to do both or just the new one will do?

Reply:
Do you mean both Back buttons? We want to attach the functionality to both. Whether you click "Back" on the drug stuck submission form page, or on the success page, you should be directed back to the top-level progress tab.

Reply:
I was wondering if the same button click triggers both the "clicks" (dumb thought). This makes sense 👍🏼